### PR TITLE
Work around the current jakarta.enterprise:jakarta.enterprise.cdi-api…

### PR DESCRIPTION
**Pull request by starksm64**
… staging issue by adding repository after sonatype-nexus-staging

Signed-off-by: Scott M Stark <starksm64@gmail.com>

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -292,6 +292,11 @@
                         <enabled>false</enabled>
                     </snapshots>
                 </repository>
+                <!-- Work around the current jakarta.enterprise:jakarta.enterprise.cdi-api staging issue -->
+                <repository>
+                    <id>cdi-api-staging</id>
+                    <url>https://oss.sonatype.org/content/repositories/central_bundles-18505</url>
```
Comment (joakime) [inline parent/pom.xml]: Would recommend adding `<name>` element on `<repository>` entry.

**Comment by joakime** [comment]
seems awfully awkward to release a pom with a temporary staging repo url.
one that becomes invalid once cdi 2.0.2 releases.
or if/when cdi has to restage 2.0.2.

have you reached out to oss.sonatype.org and asked them why that bundle upload isn't present on the `staging` group repo?

**Comment by arjantijms** [comment]
@joakime 

See here: https://issues.sonatype.org/browse/OSSRH-50614

**Comment by joakime** [comment]
``` xml
<url>https://oss.sonatype.org/content/repositories/central_bundles-18505</url>
```

Why is this PR against an old url / staged copy of `jakarta/enterprise/jakarta.enterprise.cdi-api/2.0.2`
That's not even the most recent stage of 2.0.2 according to the oss.sonatype.org list of staged repositories.

cdi-oss-multiple-bundles

I would have expected `central_bundles-18583` to be used.

**Comment by joakime** [comment]
> @joakime
> 
> See here: https://issues.sonatype.org/browse/OSSRH-50614

Cool, the issue has been fixed by the fine folks at OSSRH. (always had a pleasant experience with that group)
Looks like the `cdi-bot` permissions have been adjusted to allow deployment to the standard jakarta staging repos now. (per comments on that issue)
A restage by the cdi group's jenkins job should make things all sane again.
If that works, this PR becomes obsolete.

**Comment by starksm64** [comment]
I have gotten the api staging build working as expected and the api jars are available in the ee4j staging repos:

https://oss.sonatype.org/content/repositories/jakartaenterprise-1000
https://oss.sonatype.org/content/repositories/staging/jakarta/enterprise/jakarta.enterprise.cdi-api/2.0.2/

Closing this as now obsolete.

**Comment by arjantijms** [comment]
Fab! Thank you for the update, I’ll update Security and Faces then.

**Pull request closed without merging**